yarn.lock

```diff
@@ -1366,6 +1366,11 @@ bindings@~1.2.1:
   resolved "https://registry.yarnpkg.com/bindings/-/bindings-1.2.1.tgz#14ad6113812d2d37d72e67b4cacb4bb726505f11"
   integrity sha1-FK1hE4EtLTfXLme0ystLtyZQXxE=
 
+bluebird@^3.0.6:
```

Comment:
bluebird is a pretty bloated Promise library, that it's a shame is needed, but whatcha gonna do🤷‍♂